### DISCUSSION TITLE
#30DaysOfDataScience - Curating your Data Science Portfolio Panel

### DISCUSSION THREAD

**Discussion by carlotta94c** [microsoft/30daysof#144] [Ideas]
# Make your questions about building a data science portfolio answered ✨
As part of #30DaysOfDatascience (#15) we'll be hosting a panel next week on Microsoft Reactor youtube channel, with 3 subject matter experts sharing their tips and tricks on how they curate their data science portfolio :bar_chart: :books:!
You can learn more on the session and RSVP  [here](https://www.meetup.com/en-AU/microsoft-reactor-london/events/289108730/)! 

:bulb: But there's a **call to action** for you: to ensure the session will be the most useful as possible for you and will be able to clarify all your doubts, **please post here the questions you would like to ask** to the panel's guests. Thank you all and see you next week :smiley:

**Comment by tracynuwagaba** [comment]
Hi, I'm Tracy currently learning data science. I would love to know which data science projects I can use to optimize my github profile. I would love to work on projects as I learn. 

Please note that I know how to use github. I have used it before for web development projects before I switched to data science. I don't know how to start using my github with data science projects as a beginner. I really want to build my github profile.
